account_reconcile_compassion/models/account_reconcile_model.py

```diff
@@ -69,8 +78,33 @@ def _get_invoice_matching_query(self, st_lines_with_partner, excluded_ids):
         :return:                (query, params)
         '''
         query, params = super()._get_invoice_matching_query(st_lines_with_partner, excluded_ids)
-        bank_statement_date = self.env.context.get("bank_statement_date")
-        if params.get("aml_date_limit") and bank_statement_date:
-            date_limit = bank_statement_date - relativedelta(months=self.past_months_limit)
-            params['aml_date_limit'] = date_limit
-        return query, params
\ No newline at end of file
+        for line in st_lines_with_partner:
+            line = line[0]
```

Comment:
It's a bad practice to redefine the variable that is used in a loop. I would rather use another variable for that:
```suggestion
        for line in st_lines_with_partner:
            bank_st_line = line[0]
```